src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation.module.scss

```diff
@@ -0,0 +1,195 @@
+/* Branding */
+.navbar-openattic {
```

Comment:
Since you're removing openattic specific namings, why leaving this here?

Reply:
Ack, will change this.